src/main/java/com/customerApi/api/services/CustomerService.java

```diff
@@ -0,0 +1,110 @@
+package com.customerApi.api.services;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.customerApi.api.exception.CustomerNotFoundException;
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.repositories.CustomerRepository;
+
+@Service
+public class CustomerService {
+
+	private static final Logger logger = LoggerFactory.getLogger(CustomerService.class);
+
+	@Autowired
+	private CustomerRepository customerRepo;
+
+	public boolean addCustomerData(Customer customer) {
```

Comment:
Return customer object in response in case of success. Make sure in database ID is auto-generated, and you are not passing it in the request. If save operation via success then it will return customers with generated ID in it. And based on ID present you can see if it's success or failure. And use Optional in service response.